crates/types/protobuf/restate/deprecated_cluster.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2024 - 2025 Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+syntax = "proto3";
+
+import "restate/common.proto";
+import "restate/cluster.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+package restate.deprecated_cluster;
```

Comment:
I'm assuming that this file is a simple rename that github doesn't recognize as a rename, right?

Reply:
It's confused because we still have a file with the same `old` name (cluster.proto)